datadog_cluster_agent/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## 2.7.2 / 2023-10-26 / Agent 7.49.0
+
+***Added***
+
+* Add `API queries response status` widget to the `Datadog Cluster Agent - Overview` dashboard
```

Comment:
Please undo changes to this file and provide this sentence to the [`ddev release changelog new` ](https://datadoghq.dev/integrations-core/ddev/cli/#ddev-release-changelog-new) command